synfig-studio/src/gui/workarea.cpp

```diff
@@ -629,41 +621,34 @@ WorkArea::load_meta_data()
 	if(data.size() && (data=="0" || data[0]=='f' || data[0]=='F'))
 		set_background_rendering(false);
 
-	data=canvas->get_meta_data("guide_x");
-	get_guide_list_x().clear();
-	while(!data.empty())
-	{
-		String::iterator iter(find(data.begin(),data.end(),' '));
-		String guide(data.begin(),iter);
-	    ChangeLocale change_locale(LC_NUMERIC, "C");
-
-		if(!guide.empty())
-			get_guide_list_x().push_back(stratof(guide));
-
-		if(iter==data.end())
-			data.clear();
-		else
-			data=String(iter+1,data.end());
-	}
-	//sort(get_guide_list_x());
-
-	data=canvas->get_meta_data("guide_y");
-	get_guide_list_y().clear();
+	//for our guide to be stored wed have to store 2 floats x,y + 1 float angle in rad + a true or false
+	data=canvas->get_meta_data("guide");
+	get_guide_list().clear();
 	while(!data.empty())
 	{
 		String::iterator iter(find(data.begin(),data.end(),' '));
-		String guide(data.begin(),iter);
-	    ChangeLocale change_locale(LC_NUMERIC, "C");
-
-		if(!guide.empty())
-			get_guide_list_y().push_back(stratof(guide));
+		String guide(data.begin(),iter);//this now contains the four things
+		ChangeLocale change_locale(LC_NUMERIC, "C");
```

Comment:
Move this line to inside `if(!guide.empty()){`  block below.
It temporarily changes locale of the whole program, so it should be placed closest to its usage (and destructed as soon as possible).

Reply:
aha ok, done.